DependencyInjection/Configuration.php

```diff
@@ -37,7 +37,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->canBeUnset()
                     ->children()
-                        ->scalarNode('template')->defaultValue('knp_menu.html.twig')->end()
+                        ->scalarNode('template')->defaultValue('KnpMenuBundle::menu.html.twig')->end()
```

Comment:
is this a risk of BC break? or did the previous default simply not work and thus had to be overwritten?